include/input/tablet.h

```diff
@@ -9,12 +9,18 @@ struct seat;
 struct wlr_device;
 struct wlr_input_device;
 
+enum lab_tablet_motion_mode {
+	LAB_TABLET_MOTION_ABSOLUTE = 0,
+	LAB_TABLET_MOTION_RELATIVE,
+};
+
 struct drawing_tablet {
 	struct wlr_input_device *wlr_input_device;
 	struct seat *seat;
 	struct wlr_tablet *tablet;
 	struct wlr_tablet_v2_tablet *tablet_v2;
-	double x, y;
+	enum lab_tablet_motion_mode motion_mode;
+	double x, y, dx, dy;
```

Comment:
Note to self: no need to remember dx, dy since they are relative. May be the same applies for tilt?

Reply:
Let's keep it like this for readability.